index.js

```diff
@@ -3,7 +3,7 @@ openLinksInExternalBrowserByDefault();
 
 $(document).ready(function () {
     // translate to user-selected language
-    localize();
+    if (isNW()) { localize(); }
```

Comment:
Does not work the localization under chrome?

If I'm not wrong, the standard coding of Betaflight is to not have all in the same line.

Reply:
Localization uses the chrome.i18n API which is only available to extensions. It would of course be possible to build a shim, but since it's only used for translating strings of the version check and update, it's not of much use since that is anyway not possible to debug this way.